.github/workflows/bump-golang.yml

```diff
@@ -21,3 +24,5 @@ jobs:
           vaultRoleId: ${{ secrets.VAULT_ROLE_ID }}
           vaultSecretId: ${{ secrets.VAULT_SECRET_ID }}
           pipeline: ./.github/updatecli-bump-golang.yml
+          notifySlackChannel: "#ingest-notifications"
+          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@update-me-with-the-slack-team-to-be-poked` please look what's going on <${{ env.JOB_URL }}|here>"
```

Comment:
What team should be pinged? 
```suggestion
          messageIfFailure: ":traffic_cone: updatecli failed for `${{ github.repository }}@${{ github.ref_name }}`, `@UPDATEME` please look what's going on <${{ env.JOB_URL }}|here>"
```

Reply:
#ingest-notifications

Reply:
That's the channel but within the channel, it's possible to ping a given team with the `@team`. That's quite handy when there too many notifications in the slack channel

Reply:
For instance, currently,  we are the ones receiving those notifications in Slack:

<img width="887" alt="image" src="https://github.com/elastic/elastic-agent/assets/2871786/c8de6090-4101-4bcb-aa6c-931a96ded6e2">

Reply:
@agent-team shoud do the job